android/app/src/main/java/app/organicmaps/car/screens/download/DownloaderHelpers.java

```diff
@@ -15,9 +16,14 @@ public final class DownloaderHelpers
 {
   static final String[] WORLD_MAPS = new String[]{"World", "WorldCoasts"};
 
+  // World maps may be missing only in the F-Droid build.
+  @SuppressWarnings("ConstantConditions")
   public static boolean isWorldMapsDownloadNeeded()
```

Comment:
Is it feasible to make a more reliable check, that actually scans the presence of files in the filesystem? There should be an easy way to do this, right @vng?

Reply:
It is called `legacy`, right? :)

Reply:
@AndrewShkrob There is no any better alternative to the legacy now. Use it as an example.